app/models/team_member.rb

```diff
@@ -0,0 +1,23 @@
+class TeamMember
+  attr_accessor :user_id, :given_name, :family_name, :email, :roles
```

Comment:
```suggestion
  attr_reader :user_id, :given_name, :family_name, :email, :roles
```

should be sufficient?